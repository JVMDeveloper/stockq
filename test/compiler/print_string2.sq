print("How's it going, friend?");
print("Not bad. How about you?");
print("Terrible.");
