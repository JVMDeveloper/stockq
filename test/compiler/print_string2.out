How's it going, friend?
Not bad. How about you?
Terrible.

